### PR TITLE
setup of titanlib qc in the toolkit test with buddy check

### PR DESCRIPTION
**Pull request by vergauwenthomas**

### CODE REVIEW

**Comment by vergauwenthomas** [comment]
TODO:

- [ ]  update self
- [ ]  add colors for this type of outlier
- [ ]  set back default values
- [ ]  test qc stats + timeseries + combine all etc
- [ ]  write test

File: metobs_toolkit/settings_files/qc_settings.py [hunk 1]
```diff
@@ -41,6 +41,51 @@
...
+titan_specific_labeler = {
+    'titan_buddy_check': {
+        'ok' : [0],
+        'outl': [1]
+        },
+    
+    'titan_sct_resistant_check': {
+        'ok' : [0],
+        'outl': [1]
+        }
```
Comment (vergauwenthomas) [inline metobs_toolkit/settings_files/qc_settings.py:86]: The numeric output can be 0,1,11,12 and -999

File: metobs_toolkit/dataset.py [hunk 2]
```diff
@@ -1215,6 +1217,77 @@ def apply_quality_control(
         self._qc_checked_obstypes = list(set(self._qc_checked_obstypes))
         self.outliersdf = self.outliersdf.sort_index()
 
+    def apply_titan_buddy_check(self, obstype='temp'):
+        # TODO: what if elevation is not availabl
```
Comment (vergauwenthomas) [inline metobs_toolkit/dataset.py:1221]: Add docstring

File: metobs_toolkit/dataset.py [hunk 3]
```diff
@@ -1215,6 +1217,77 @@ def apply_quality_control(
...
+
```
Comment (vergauwenthomas) [inline metobs_toolkit/dataset.py:1222]: Test if altitude is available

File: metobs_toolkit/dataset.py [hunk 4]
```diff
@@ -1215,6 +1217,77 @@ def apply_quality_control(
...
+        # add colors for this type of outlier
+        # set back default values
+        # test qc stats + timeseries + combine all etc
+        # write test
+
+
+        return obsdf, outliersdf
+    
+    def apply_titan_sct_resistant_check(self, obstype='temp'):
+        # TODO: what if elevation is not availabl
```
Comment (vergauwenthomas) [inline metobs_toolkit/dataset.py:1257]: add docstring

File: metobs_toolkit/dataset.py [hunk 5]
```diff
@@ -1215,6 +1217,77 @@ def apply_quality_control(
...
+
```
Comment (vergauwenthomas) [inline metobs_toolkit/dataset.py:1258]: check if altitude is available

File: metobs_toolkit/dataset.py [hunk 6]
```diff
@@ -1215,6 +1217,77 @@ def apply_quality_control(
...
+                    obstypes=obstype, ordered_checknames="titan_buddy_check"
+                ),
+            ],
+            ignore_index=True,
+        )
+
+        # add colors for this type of outlier
+        # set back default values
+        # test qc stats + timeseries + combine all etc
+        # write test
```
Comment (vergauwenthomas) [inline metobs_toolkit/dataset.py:1287]: Remove

File: metobs_toolkit/settings_files/qc_settings.py (hunk 1 above)
Comment (MichielVieijra) [reply metobs_toolkit/settings_files/qc_settings.py:86]: 11, 12 and -999 are for observations that are not checked, so only 0 and 1 are needed for 'ok' and 'outlier' labels

**Review by MichielVieijra — COMMENTED** [review]
I made a few changes based on the comments and now the code is ready for me to be merged.

**Review by MichielVieijra — APPROVED** [review]

**Pull request merged**